plugins/modules/ec2_group.py

```diff
@@ -159,7 +176,10 @@
         proto:
             type: str
             description:
-            - The IP protocol name (C(tcp), C(udp), C(icmp), C(icmpv6)) or number (U(https://en.wikipedia.org/wiki/List_of_IP_protocol_numbers))
+            - The IP protocol name (C(tcp), C(udp), C(icmp), C(icmpv6)) or
+            - number (U(https://en.wikipedia.org/wiki/List_of_IP_protocol_numbers))
+            - When using a C(proto: icmp) or C(proto: icmpv6), you can pass the
+            - C(icmp_type) and C(icmp_code) parameters instead of C(from_port) and C(to_port).
```

Comment:
```suggestion
            - When using a I(proto=icmp) or I(proto=icmpv6), you can pass the
              I(icmp_type) and I(icmp_code) parameters instead of I(from_port) and I(to_port).
```